src/main/java/GameField.java

```diff
@@ -49,21 +49,21 @@ private boolean intersectsObjects(Rectangle rectangle, Collection<GameObject> ob
         return false;
     }
 
-    private boolean outOfField(Rectangle rectangle){
-        return !field.includes(rectangle);
+    private boolean isOutOfGameField(Rectangle objectBody){
+        return !field.includes(objectBody);
     }
 
     private boolean canMove(GameObject gameObject){
         Point speed = gameObject.getSpeed();
-
+        
         movedBody.setRectangle(gameObject.getBody());
         movedBody.shift(speed);
 
-        if (outOfField(movedBody)){
+        if (isOutOfGameField(movedBody)){
```

Comment:
add extra space between ")" and "{". same everywhere. 

To avoid such issues apply CTRL+ALT+SHIFT+L to all project files in a separate commit with message "CTRL+ALT+SHIFT+L"